en_ewt-ud-dev.conllu

```diff
@@ -6229,7 +6229,7 @@
 11	to	to	PART	TO	_	12	mark	12:mark	_
 12	control	control	VERB	VB	VerbForm=Inf	10	acl	10:acl:to	_
 13	who	who	PRON	WP	PronType=Int	18	parataxis	18:parataxis	_
-14	when	when	ADV	WRB	PronType=Int	20	mark	20:mark	_
+14	when	when	SCONJ	WRB	PronType=Int	20	mark	20:mark	_
```

Comment:
coordinated WH-words: should all be `SCONJ`?

Reply:
if they are advmod (which functionally they are, notwithstanding `conj`), they should be ADV by the current guidelines.

Reply:
Sorry, I take it back - upon another look I think you're right about SCONJ here.